doc/reference/dune/menhir.rst

```diff
@@ -44,6 +44,13 @@ Therefore, if this is flag is passed to Menhir, Dune will know to introduce a
   :doc:`/reference/boolean-language`. This field was introduced in version 3.0
   of the Menhir extension.
 
+- ``(messages <name>)`` Specify ``<name>.messages`` as a "messages" file for
+  advanced error reporting using Menhir. Dune will ask Menhir to generate
+  ``<name>.ml`` from it, as well as to keep it up-to-date by using the various
+  facilities of Menhir for this purpose (see the Menhir manual for
+  details). This option is available starting with version 3.1 of the Menhir
+  extension.
```

Comment:
I wonder, would it make sense to disable that in `--release`? From what I understand `.messages` only really makes sense when developing, do debug your parsers.